var/spack/repos/builtin/packages/py-cython/package.py

```diff
@@ -45,6 +47,17 @@ class PyCython(PythonPackage):
     depends_on("py-setuptools", type=("build", "run"))
     depends_on("gdb@7.2:", type="test")
 
+    @property
+    def prefix(self):
+        prefix = super(PyCython, self).prefix
+        if os.path.isdir(os.path.join(prefix, "local", "bin")) and \
+                not os.path.isdir(os.path.join(prefix, "bin")):
+            prefix.bin = os.path.join(prefix, "local", "bin")
+        if os.path.isdir(os.path.join(prefix, "local", "lib")) and \
+                not os.path.isdir(os.path.join(prefix, "lib")):
+            prefix.lib = os.path.join(prefix, "local", "lib")
```

Comment:
This won't help if you're on an OS that uses `lib64` instead of `lib`.

Reply:
You are right, added an additional `elif` clause to cover this. Don't like any of these solutions, but the solutions you mentioned above aren't available in the authoritative spack repo yet.

Reply:
Note that you'll also need to do what you're doing here for every package in Spack...

Reply:
I know ... my strategy is to have this fixed temporarily (because it's the only package in our fairly comprehensive software stack that has this problem) and then get the clean solution from spack once that is available.